### PR TITLE
Move mixins to global space

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
This PR is awesome.

It will lead to better integration of resource-loader and all my plugins. We'll get proper mixins declaration in `d.ts` files!

This is initial demo - I expose only DisplayObject. More stuff will be exposed later when we discuss this one.

Many mixins will be exposed for canvas packages.

### CODE REVIEW

File: packages/interaction/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+declare type InteractionTrackingData = any;
+
+declare type Cursor = 'auto'
```
Comment (cursedcoder) [inline packages/interaction/global.d.ts]: sorcery

**Comment by cursedcoder** [comment]
I'll need to test this locally, will let you know tomorrow :)

File: packages/interaction/global.d.ts (hunk 1 above)
Comment (ivanpopelyshev) [reply packages/interaction/global.d.ts]: Those particular lines were moved it from globals. They will be removed as soon as someone converts interaction to TS :)

File: packages/accessibility/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+declare namespace PixiGlobalMixins
```
Comment (bigtimebuddy) [inline packages/accessibility/global.d.ts]: Why not just call this namespace `PIXI`?

Comment (ivanpopelyshev) [reply packages/accessibility/global.d.ts]: I agree to change the name to whatever you want except `PIXI`: this thing does not actually exists, its only a TS hack. I dont want it to collide with actual global namespace if we'll have any. 

Also people will type `PIXI.` in IDE, see stub interfaces and they wont understand whats going on.

Comment (bigtimebuddy) [reply packages/accessibility/global.d.ts]: Mixins?

Comment (cursedcoder) [reply packages/accessibility/global.d.ts]: GlobalMixins sounds better IMO

Comment (ivanpopelyshev) [reply packages/accessibility/global.d.ts]: GlobalMixins it is.

**Comment by bigtimebuddy** [comment]
Good one!

**Pull request merged**